### PR TITLE
PFW-1520: Add new message `Preload to MMU` to po files

### PR DESCRIPTION
**Pull request by 3d-gussner**
Initial translation

Has to be reviewed by translators

- [x] CS @DRracer 
- [x] DE
- [x] ES
- [x] FR
- [x] HR @Prime1910 
- [x] HU @AttilaSVK @Hauzman
- [x] IT @wavexx 
- [x] NL @vintagepc @stelgenhof
- [x] NO @OS-kar @trondkla
- [x] PL
- [x] RO @leptun @Hauzman
- [x] SK @ingbrzy @shatter136
- [x] SV @Painkiller56

### CODE REVIEW

**Comment by Prime1910** [comment]
Croatian looks good :)

**Comment by trondkla** [comment]
(_no) Norwegian looks good. Would "Forhåndslast MMU" be better Norwegian @OS-kar ?

**Comment by arekm** [comment]
For Polish I see "ł" used in po file, so

Zaladuj do MMU -> Załaduj do MMU

**Comment by shatter136** [comment]
In SK, the translation is now Czech, so correct should perhaps be "Predzásobenie"

**Comment by AttilaSVK** [comment]
Hi! Hungarian should be "Betöltés az MMU-ba"

3d-gussner ***@***.***> ezt írta (időpont: 2023. máj. 22., H,
9:44):

**Comment by leptun** [comment]
RO looks good

**Comment by 3d-gussner** [comment]
> (_no) Norwegian looks good. Would "Forhåndslast MMU" be better Norwegian @OS-kar ?

Issue is that it limited to 17 chars

**Comment by 3d-gussner** [comment]
> Betöltés az MMU-ba

It is 18 char but we are limited to 17 what is the best to shorten it?

**Comment by AttilaSVK** [comment]
Just leave out the hyphen: Betöltés az MMUba

3d-gussner ***@***.***> ezt írta (időpont: 2023. máj. 22., H,
10:34):

**Review by shatter136 — APPROVED** [review]
Ok now

**Comment by SNDR0** [comment]
In italian will be better "Caricamento MMU" , if we consider the mmu like a magazine.. or "Precaricamento MMU" if you want to translate from English...

**Comment by 3d-gussner** [comment]
> In italian will be better "Caricamento MMU" , if we consider the mmu like a magazine.. or "Precaricamento MMU" if you want to translate from English...

@SNDR0 :bow:  Thanks for the suggestion I will forward that to the internal translators.

File: lang/po/Firmware_nl.po [hunk 1]
```diff
@@ -2556,6 +2556,11 @@ msgstr ""
 "De firmwareversie van de MMU is niet compatibel met de firmware van de "
 "printer. Update naar versie 2.1.9."
 
+#. MSG_PRELOAD_TO_MMU c=17
+#: ../../Firmware/messages.cpp:55 ../../Firmware/ultralcd.cpp:5227
+msgid "Preload to MMU"
+msgstr "Preload naar MMU"
```
Comment (stelgenhof) [inline lang/po/Firmware_nl.po]: I would suggest to change it to "Voorladen in MMU".

**Review by stelgenhof — CHANGES_REQUESTED** [review]

**Comment by Painkiller56** [comment]
Swedish looks good.

**Comment by 3d-gussner** [comment]
> In italian will be better "Caricamento MMU" , if we consider the mmu like a magazine.. or "Precaricamento MMU" if you want to translate from English...

@SNDR0 :bow:  Thanks for the suggestion I will forward that to the internal translators.

**Comment by wavexx** [comment]
> In italian will be better "Caricamento MMU" , if we consider the mmu like a magazine.. or "Precaricamento MMU" if you want to translate from English...

From the context, I'd go for "Precarica MMU", and maybe also change "Carica tutti" to "Precarica tutti" for consistency. What do you think @SNDR0 ?

**Comment by SNDR0** [comment]
@wavexx 
Wonderful... Nothing to add...

**Comment by trfunk** [comment]
I would suggest "In MMU laden" for german, it's similar to the "load to nozzle" msg then and "vorladen" isn't really used.

**Comment by 3d-gussner** [comment]
> I would suggest "In MMU laden" for german, it's similar to the "load to nozzle" msg then and "vorladen" isn't really used.

Thanks! 
Dankeschön, habe Deine Übersetzung übernommen. Gut gesehen das andere ähnlich sind.

**Comment by 3d-gussner** [comment]
> and maybe also change "Carica tutti" to "Precarica tutti" for consistency. 

Issue is that `Load all` is used twice and other translations are still based on that term.
I will ask if we should change `Load all` to `Preload all`.

**Comment by SNDR0** [comment]
Last question... Isn't better write "Preload MMU"?

**Comment by DRracer** [comment]
@SNDR0 "Preload MMU" would also make sense, but IMO "Preload to MMU" is more explicit, that we want to put the filament into the MMU (and not into the printer)

**Comment by Janys** [comment]
@3d-gussner @DRracer Czech: "Zavést do MMU"

**Comment by 3d-gussner** [comment]
Thanks to everyone for fast review :hugs:

**Review by gudnimg — APPROVED** [review]

**Pull request merged**